### PR TITLE
Adding Oregon convenience store (Dari Mart) with about 40 stores.

### PR DESCRIPTION
**Pull request by b-jazz**

### CODE REVIEW

File: brands/shop/convenience.json [hunk 1]
```diff
@@ -507,6 +507,16 @@
       "shop": "convenience"
     }
   },
+  "shop/convenience|Dari Mart": {
+    "tags": {
+      "brand": "Dari Mart",
+      "brand:website": "https://www.darimart.com/",
```
Comment (mds08011) [inline brands/shop/convenience.json]:
```suggestion
```

Comment (mds08011) [reply brands/shop/convenience.json]: We haven't been including website tags with other entries so I recommend removing this.

Comment (b-jazz) [reply brands/shop/convenience.json]: It seems like a perfect way to add some extra data to the objects, but I think following convention is more important, so I've removed the line.

Comment (mds08011) [reply brands/shop/convenience.json]: Agreed.  I’d say this is worth discussing in the #poi channel of the OSM-US Slack.

**Comment by mds08011** [comment]
Thank you!

**Pull request merged**